packages/medusa/src/models/cart.ts

```diff
@@ -390,6 +391,13 @@ export class Cart extends SoftDeletableEntity {
   gift_card_total?: number
   gift_card_tax_total?: number
 
+  @AfterLoad()
+  private afterLoad(): void {
```

Comment:
comment: We can now bring this back, as we are no longer using the query strategy.